docs/docs/tutorial-gce.md

```diff
@@ -1,6 +1,11 @@
 ---
 title: Mesos-DNS on GCP
+
+redirect_to:
+- http://mesosphere.github.io/mesos-dns/
+
 ---
+<!-- This is deprecated -->
```

Comment:
OK we can open a new PR to add links for a proper tutorial. In the meantime
we should avoid leaving comment blocks in the code like this. Please remove
and squash the PR.

On Tue, Feb 2, 2016 at 7:06 PM, Joel Hamill <notifications@github.com>
wrote:

> In docs/docs/tutorial-gce.md
> <https://github.com/mesosphere/mesos-dns/pull/389#discussion_r51658671>:
>
> >  ---
> > +<!-- This is deprecated -->
>
> @jdef <https://github.com/jdef> There's talk of possibly rewriting the
> tuturial, without GCE - so left in. Feel free to remove.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/mesos-dns/pull/389/files#r51658671>.
>